style-guide.rst

```diff
@@ -49,7 +47,7 @@ Headings
    * - Underlines with other characters, for example, ``"``, ``#`` or ``*``
```

Comment:
In my experience, whenever a H5 is needed the page is most likely not well structured. I don't think we should encourage it.